src/main/java/com/iexec/blockchain/command/task/contribute/TaskContributeArgs.java

```diff
@@ -16,21 +16,19 @@
 
 package com.iexec.blockchain.command.task.contribute;
 
-import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.iexec.blockchain.command.generic.CommandArgs;
-import lombok.Builder;
+import lombok.AllArgsConstructor;
 import lombok.Data;
 
 @Data
-@Builder
+@AllArgsConstructor
 public class TaskContributeArgs implements CommandArgs {
 
-    @JsonIgnore
-    private String chainTaskId;
-    private String resultDigest;
-    private String workerpoolSignature;
-    private String enclaveChallenge;
-    private String enclaveSignature;
+    private final String chainTaskId;
+    private final String resultDigest;
+    private final String workerpoolSignature;
+    private final String enclaveChallenge;
+    private final String enclaveSignature;
```

Comment:
Doesn't putting fields as final cause the exception `Cannot construct instance of com.iexec..... (no Creators, like default constructor, exist)`

Reply:
I cleaned that part but not I'm not having such error at runtime.